pkg/storage/s3.go

```diff
@@ -338,7 +338,10 @@ func (rs *S3Storage) ReadFile(ctx context.Context, file string) ([]byte, error)
 
 	result, err := rs.svc.GetObjectWithContext(ctx, input)
 	if err != nil {
-		return nil, errors.Trace(err)
+		er := errors.Trace(err)
```

Comment:
`Annotatef` already record stack info. so this line can be removed

Reply:
Thanks for your kind review, i will update the code later